src/EnergyPlus/DXCoils.cc

```diff
@@ -11174,10 +11174,8 @@ void CalcDXHeatingCoil(EnergyPlusData &state,
             // Calculate defrost adjustment factors depending on defrost control type
             if (thisDXCoil.DefrostControl == StandardRatings::HPdefrostControl::Timed) {
                 FractionalDefrostTime = thisDXCoil.DefrostTime;
-                if (FractionalDefrostTime > 0.0) {
-                    HeatingCapacityMultiplier = 0.909 - 107.33 * OutdoorCoildw;
-                    InputPowerMultiplier = 0.90 - 36.45 * OutdoorCoildw;
-                }
+                HeatingCapacityMultiplier = 0.909 - 107.33 * OutdoorCoildw;
+                InputPowerMultiplier = 0.90 - 36.45 * OutdoorCoildw;
```

Comment:
Enable the capacity and input power degradation coefficients to be applied even when the defrost time period fraction is 0. In other words, degrade the performance of the coil due to frost built-up even if defrost is not performed.